modules/products/bitbucket/nfs/variables.tf

```
@@ -0,0 +1,22 @@
+variable "product" {
+  description = "The product that NFS will be created for."
+  type        = string
+}
+
+variable "namespace" {
+  description = "Kubernetes namespace to install Bitbucket."
```

```suggestion
  description = "Kubernetes namespace to install NFS server."
```

Shouldn't we always install NFS to the namespace?